riscv/insns/prefetch_i.h

```diff
@@ -0,0 +1,2 @@
+require_extension(EXT_ZICBOP);
```

Comment:
Delete this line since this instruction is a hint instruction and *must not* generate illegal instruction trap.

Reply:
You can optionally stop prefetching if `EXT_ZICBOP` is not enabled. Even so, we *must not generate any* trap here (even `prefetch.i 0(zero)` must be safe).

Reply:
OK. I'll fix this.